### PR TITLE
fix e2e tests for 21-4142

### PR DESCRIPTION
**Pull request by rhasselle-oddball**
## Summary

- Fix e2e tests for 21-4142
- Fix alternate signer passing too long of a value to the backend

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team-forms#652

## Testing done

- Browser
- Unit
- E2E
- PDF

## Screenshots

No change

## What areas of the site does it impact?

e2e tests

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/maximal-test.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
       "preparerTitle": "Super title",
       "preparerOrganization": "Ultra org",
       "courtAppointmentInfo": "court info here",
-      "relationshipToVeteran": "Alternate Signer"
+      "relationshipToVeteran": "Alternate signer (a person certified to file a claim forms for the Veteran or non-Veteran) "
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/maximal-test.json:21]: updating mock data to match how it is sent today

File: src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/maximal-test.json [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
     },
     "limitedConsent": "limitations, limitations, limitations, limitations, limitations, limitations, ",
     "providerFacility": [
       {
         "providerFacilityName": "Hospital 1",
-        "conditionsTreated": "conditions here, conditions here, conditions here, conditions here, conditions here, conditions here, conditions here, conditions here",
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/maximal-test.json:27]: this is one of the things e2e was failing on. max limit of 75 chars here.

File: src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/minimal-test.json [hunk 3]
```diff
@@ -13,12 +13,20 @@
       "fullName": { "first": "veteran", "last": "name" }
     },
     "preparerIdentification": {
-      "relationshipToVeteran": "I am the service member/Veteran"
+      "relationshipToVeteran": "I am the Veteran",
+      "preparerFullName": {},
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-4142/tests/e2e/fixtures/data/minimal-test.json:17]: updating to match how data is sent today

**Review by rhasselle-oddball — COMMENTED** [review]
notes

**Review by tlei123 — APPROVED** [review]
LGTM

**Pull request merged**